webClient/src/app/editor/code-editor/monaco/monaco.service.ts

```diff
@@ -425,12 +425,31 @@ export class MonacoService implements OnDestroy {
               fileName: fileContext.model.fileName, ...(fileDirectory && {fileDirectory: fileDirectory}) }
           });
           saveRef.afterClosed().subscribe(result => {
-          if (result) {
-            this.editorControl.saveBuffer(fileContext, result).subscribe(() => obs.next('Save'));
-          } else {
-            obs.next('Cancel');
-          }
-          
+            // Check if file already exists at destination
+            this.editorControl.getFileMetadata(result.directory + '/' + result.fileName).subscribe(r => {
+              const title = `"${result.fileName}" already exists. Do you want to replace it?`;
```

Comment:
In some areas, we don't use quotation marks for file name ie
```
this.snackBar.open(`${_activeFile.name} has been saved!`, 'Close', { duration: MessageDuration.Short, panelClass: 'center' });
```

```
this.snackBar.open(`${_activeDataset.name} could not be saved, feature not yet implemented.`, 
                           'Close', { duration: MessageDuration.Long,   panelClass: 'center' });   
```
so let's keep it consistent

Reply:
Done